### PR TITLE
CMP-1253 updated CO supported profiles table

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.6+

Issue:
[CMP-1253](https://issues.redhat.com/browse/CMP-1253)

Link to docs preview:
[Supported compliance profiles](http://file.rdu.redhat.com/antaylor/06-29-22/CMP-1253/security/compliance_operator/compliance-operator-supported-profiles.html)
[Release notes](http://file.rdu.redhat.com/antaylor/06-29-22/CMP-1253/security/compliance_operator/compliance-operator-release-notes.html#compliance-operator-release-notes-0-1-52)

Additional information:
This pull request adds new FedRAMP high profiles and lists them under new features and enhancements in the release notes. I've also added a known issue with this version of the Compliance Operator.

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@jhrozek  I added the known issue from [bz2092913](https://bugzilla.redhat.com/show_bug.cgi?id=2092913), let me know what you think. Also, is there any workaround we can document?

**Comment by jhrozek** [comment]
> @jhrozek I added the known issue from [bz2092913](https://bugzilla.redhat.com/show_bug.cgi?id=2092913), let me know what you think. Also, is there any workaround we can document? Or is it just `oc delete pod <podname>`?

Yes, the workaround is to delete the scan's pods manually, e.g.:
```
oc delete pods -lcompliance.openshift.io/scan-name=ocp4-cis
```

**Comment by jhrozek** [comment]
> @jhrozek I added the known issue from [bz2092913](https://bugzilla.redhat.com/show_bug.cgi?id=2092913), let me know what you think. Also, is there any workaround we can document? Or is it just `oc delete pod <podname>`?

btw the tests look good to me!

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -33,6 +38,17 @@ The following advisory is available for the OpenShift Compliance Operator 0.1.52
 
 * Previously, the Machine Config Operator used `base64` instead of `url-encoded` code in the latest release, causing Compliance Operator remediation to fail. Now, the Compliance Operator checks encoding to handle both `base64` and `url-encoded` Machine Config code and the remediation applies correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082431[*BZ#2082431*])
 
+[id="compliance-operator-0-1-52-known-issue"]
+== Known issue
+
+* When `"debug":true` is set within the scansettingbinding, the pods generated by scansettingbinding are not removed when the scansettingbinding is deleted. As a workaround, the following command will delete the remaining pods:
```
Comment (stevsmit) [inline security/compliance_operator/compliance-operator-release-notes.adoc:44]: In our docs, the word "scansettingbinding" is currently written several ways: 

`ScanSettingBinding` (with backticks + capitalized)
scansettingbinding (like you have)
ScanSettingBinding (no backticks) 

This might be context dependent, but I'm not sure personally. In your example, is it like...

When `"debug":true` is set within the `ScanSettingBinding` object. . ." ? 

tl;dr--just bringing this up for consideration.

**Comment by sheriff-rh** [comment]
Rebased and pushed to resolve build errors...
